src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -159,7 +159,7 @@ const CCPTabPanel = ({ eventId }: { eventId: string }) => {
       });
 
       const updatedCCPsList = collectionPointsByEvent.filter(
-        (ccp) => ccp.id !== (selectedCCP as CCP).id
+        (ccp) => ccp.id !== deleteCollectionPoint
```

Comment:
The backend has been updated to return the `id` of the deleted entity, so we're updating this to check using that returned `id`.